.github/workflows/build-packages.yml

```diff
@@ -331,7 +345,8 @@ jobs:
   #26
   symbiyosys-linux:
```

Comment:
You should probably build SymbiYosys against both these.

Reply:
Done.